frontend/src/container/GridGraphLayout/Graph/index.tsx

```diff
@@ -77,9 +85,15 @@ function GridCardGraph({
 	const { dashboards } = useSelector<AppState, DashboardReducer>(
 		(state) => state.dashboards,
 	);
-	const [selectedDashboard] = dashboards;
-	const selectedData = selectedDashboard?.data;
-	const { variables } = selectedData;
+
+	let selectedDashboard: Dashboard | undefined;
```

Comment:
make a util for this one which give selectedDashboard for this one

Reply:
Okay